### PR TITLE
Add knowledge graphs to data

### PR DESCRIPTION
**Pull request by rajammanabrolu**
Adding knowledge graph per step and one step look ahead for valid action at each step. `symtables/*.out` contains decompiled attributes/properties for all games which is read when collecting data. `symtables/readable_tables.txt` contains a list of games with human readable attributes/properties.

### CODE REVIEW

File: build_dataset.py [hunk 1]
```diff
@@ -141,26 +207,136 @@ def build_dataset():
         candidate_actions = act_gen.generate_actions(interactive_objs)
         diff2acts = find_valid_actions(env, state, candidate_actions)
 
+        valid_actions = diff2acts.values()
+
+        for v in valid_actions:
```
Comment (mhauskn) [inline build_dataset.py]: Could we include the 1-step exploration in a different file? I'm focused on trying to make sure things work just along the walkthrough trajectories for now.

File: build_dataset.py [hunk 2]
```diff
@@ -8,6 +8,70 @@
...
+from glob import glob
+
+
+MOVE_ACTIONS = 'north/south/west/east/northwest/southwest/northeast/southeast/up/down/enter/exit'.split('/')
+
+with open('symtables/readable_tables.txt', 'r') as f:
+    readable = [str(a).strip() for a in f]
+
+attribues = {}
+for gn in readable:
```
Comment (mhauskn) [inline build_dataset.py]: Can we encapsulate this into a function?

**Comment by mhauskn** [comment]
Thank you Raj for the PR! I'm wondering if we can minimize changes to the current code by saving the 1-step valid-action exploration for a later PR (or splitting functionality into a separate file). The reason is that I'd like to concentrate on making sure we have got things right for the states along the walkthrough first.

Also I apologize because I think there are now a few merge conflicts as a result of the commit I made. I'm happy to help add the functionality for graph diff if you'd like.

**Comment by rajammanabrolu** [comment]
Updated to have walkthrough only (and the other changes requested) + readme update, look at branch onestep in my fork for the one step. I'll have a separate PR for that later.

**Review by mhauskn — APPROVED** [review]

**Pull request merged**